packages/core/src/server/templates/html.ts

```diff
@@ -1,17 +1,21 @@
 import { Template } from "../../types";
 
-const template: Template = ({ html, scriptTags, linkTags }) => `<!doctype html>
+const template: Template = ({ html, frontity, head }) => `<!doctype html>
     <html>
-      <head>
+      <head ${head.htmlAttributes}>
```

Comment:
This line seems wrong. Shouldn't those attributes be rendered inside `<html>`?

Reply:
You're right.

Reply:
Fixed

Reply:
Uh, I missed that before approving the PR. Nice catch, @orballo.